deephaven/vscode-deephaven
# feat: Added PanelRegistry so panels can be re-used globally

Panels are now stored in a global registry instead of being scoped to DhcService instance. This allows re-use of panels after re-connecting.

**Testing**
- Run a script
- Restart the server
- Re-run the script. Should re-use the open panel

resolves #3
